File: Problemset/1389-Create Target Array in the Given Order/Simulation.rb
# @param {Integer[]} nums
# @param {Integer[]} index
# @return {Integer[]}
def create_target_array(nums, index)
    target = Array.new

    for i in 0...nums.length
        target.insert(index[i], nums[i])
    end

    return target
end

File: Problemset/1389-Create Target Array in the Given Order/Simulation.rs
impl Solution {
    pub fn create_target_array(nums: Vec<i32>, index: Vec<i32>) -> Vec<i32> {
        let mut target = Vec::new();

        for i in 0..nums.len() {
            target.insert(index[i] as usize, nums[i]);
        }

        target
    }
}
